packages/lazy-fn-vite-plugin/tsconfig.json

```
@@ -1,4 +1,4 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["src", "tests", "vite.config.ts"]
+  "include": ["src", "tests/transpileSource.test.ts", "vite.config.ts"]
```

This kind of tsconfig is error prone since user has to manually include each future tests file.

Adding a match to `exclude` option targetting new `tests/sources` folder avoids this scenario .

https://www.typescriptlang.org/tsconfig/#exclude

> Specifies an array of filenames or patterns that should be skipped when resolving [include](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#include).

```suggestion
  "include": ["src", "tests", "vite.config.ts"],
  "exclude": ["tests/sources"]
```